package.json

```diff
@@ -26,15 +26,22 @@
     "lang:refresh": "node scripts/refresh-language.js",
     "csl:refresh": "node scripts/update-csl.js",
     "reveal:build": "node scripts/update-reveal.js",
-    "wp:dev": "cross-env NODE_ENV=development webpack --progress --colors",
-    "wp:prod": "cross-env NODE_ENV=production webpack --progress --colors",
-    "watch:gulp": "gulp watch",
-    "watch:wp": "cross-env NODE_ENV=development webpack --progress --colors --watch",
-    "watch": "npm-run-all --parallel watch:*",
     "test": "mocha",
     "test-gui": "node scripts/test-gui.js",
     "lint": "eslint --ext js,vue -f html -o eslint_report.htm source resources"
   },
+  "electronWebpack": {
+    "main": {
+      "sourceDirectory": "source/main"
+    },
+    "renderer": {
+      "sourceDirectory": "source/renderer",
+      "template": "source/renderer/assets/index.htm"
+    },
+    "whiteListedModules": [
+      "mermaid"
```

Comment:
This line is responsible for inlining dependencies that makes possible to reduce the shipped binary size.